bokeh/server/session.py

```diff
@@ -214,7 +213,7 @@ def _handle_patch(self, message, connection):
         self._current_patch = message
         self._current_patch_connection = connection
         try:
-            message.apply_to_document(self.document)
+            message.apply_to_document(self.document, self.id)
```

Comment:
a minor detail but I think you could use simply `self` rather than `self.id` as long as you then test `getattr(event, 'setter_id', None) is self` (`is` rather than `==`).